srcpkgs/code-minimap/template

```diff
@@ -0,0 +1,16 @@
+# Template file for 'code-minimap'
+pkgname=code-minimap
+version=0.6.4
```

Comment:
Version 0.6.7 has been released as well and seems to work just fine, so it might be a good idea to update this PR as well